### ISSUE TITLE
Unclear licensing

### ISSUE THREAD

**Issue by ViRuSTriNiTy** [HTMLElements/smart-webcomponents#95]
Hi there,

I am currently evaluating your component package for use in a commercial application. I tried to figure out the licensing but I am a bit confused. The page [[Community & Professional/Enterprise](https://www.htmlelements.com/docs/community-and-enterprise/)](https://www.htmlelements.com/docs/community-and-enterprise) states

> Smart comes in two forms: Community (**free, including production use**) ...

but then, when installing the npm package via ...

```
npm i smart-webcomponents-community
```

... the file [EULA.pdf](https://github.com/HTMLElements/smart-webcomponents/blob/master/EULA.pdf) is put into `node_modules/smart-webcomponents-community/EULA.pdf` which states the contrary ....

> **You must acquire a license for the Software** directly from the Author or its authorized resellers. If your website or product has no commercial intent and does not generate any direct or indirect revenue you may acquire a license without paying a fee. **For all other cases you must purchase a commercial license of the Software.**

To me it seems that commercial use of he community edition is prohibited without having a paid license. Is that true?

So lonG

**Comment by bmarkov** [comment]
The community license https://www.htmlelements.com/license/ includes 4 components - Table, Tabs, Tree and Menu. These are free to use.

**Comment by ViRuSTriNiTy** [comment]
@bmarkov Correct, thanks, figured that out myself already, but why is there an EULA.pdf in the community package stating otherwise? Is it perhaps a mistake in the packaging?

**Comment by bmarkov** [comment]
It includes the EULA because the community version is not an open source version.

**Comment by ViRuSTriNiTy** [comment]
Ok thanks, question answered, closed.
